netstandard/Elmish.fsproj

```diff
@@ -8,7 +8,9 @@
   <ItemGroup>
     <Compile Include="../src/prelude.fs" />
     <Compile Include="../src/cmd.fs" />
+    <Compile Include="../src/ring.fs" />
     <Compile Include="../src/program.fs" />
+    <Compile Include="meta.fs" />
```

Comment:
I can probably make do w/o this, we'll see if just referencing the ring source from the test project makes more sense that this jumping through the hoops.